client/src/util/centroid.js

```diff
@@ -0,0 +1,49 @@
+import quantile from "./quantile";
+
+/*
+  Centroid coordinate calculation
+*/
+const calcMeanCentroid = (world, annoName, annoValue, layoutDimNames) => {
+  const centroid = [0, 0, 0];
```

Comment:
I'd prefer to use an object `{ centroidX: 0, centroidY: 0, count: 0 }` or at least separate [x, y] and count.

Reply:
calcMeanCentroid uses an array because the function to convert an xy point to screen space asks for an array for its parameter.  Also for the moment calcMeanCentroid is unused.

Reply:
You could do the calculations using an object and then transform to an array on return. 

`return [centroid.X, centroid.Y]`